loglist/logfilter.go

```diff
@@ -39,22 +39,35 @@ func (ll *LogList) ActiveLogs() LogList {
 }
 
 // Compatible creates a new LogList containing only logs of original LogList
-// compatible with cert-chain provided.
-// Cert-chain is expected to be full: ending with CA-cert.
+// compatible with cert-chain provided. Treat Logs as compatible when their
+// root info is missing.
+// Cert-chain is expected to be full or empty: ending with CA-cert or have
+// len 0.
```

Comment:
I found this a bit confusing on first read -- how about something like:
```golang
// Compatible creates a new LogList containing only the logs of the original LogList
// that are compatible with the provided cert-chain, according to the passed in 
// collection of per-log roots.  Logs that are missing from the collection are treated
// as always compatible and included, even if an empty cert chain is passed in.
```

Reply:
Inserted, thanks!